hoodie-client/src/main/java/com/uber/hoodie/func/LazyInsertIterable.java

```diff
@@ -112,38 +95,43 @@ protected void start() {
   }
 
   private List<WriteStatus> handleWrite(
-      final BufferedIterator<T, HoodieRecord<T>> bufferedIterator) {
+      final BufferedIterator<HoodieRecord<T>, Optional<IndexedRecord>> bufferedIterator) {
     List<WriteStatus> statuses = new ArrayList<>();
     while (bufferedIterator.hasNext()) {
-      final BufferedIterator.BufferedIteratorPayload<HoodieRecord<T>> payload = bufferedIterator
+      final HoodieRecordBufferedIteratorPayload payload = (HoodieRecordBufferedIteratorPayload) bufferedIterator
           .next();
-
+      final HoodieRecord insertPayload = (HoodieRecord) payload.getInputPayload();
       // clean up any partial failures
-      if (!partitionsCleaned.contains(payload.record.getPartitionPath())) {
+      if (!partitionsCleaned
+          .contains(insertPayload.getPartitionPath())) {
         // This insert task could fail multiple times, but Spark will faithfully retry with
         // the same data again. Thus, before we open any files under a given partition, we
         // first delete any files in the same partitionPath written by same Spark partition
         HoodieIOHandle.cleanupTmpFilesFromCurrentCommit(hoodieConfig, commitTime,
-            payload.record.getPartitionPath(), TaskContext.getPartitionId(), hoodieTable);
-        partitionsCleaned.add(payload.record.getPartitionPath());
+            insertPayload.getPartitionPath(),
+            TaskContext.getPartitionId(), hoodieTable);
+        partitionsCleaned.add(insertPayload.getPartitionPath());
       }
 
       // lazily initialize the handle, for the first time
       if (handle == null) {
         handle = new HoodieCreateHandle(hoodieConfig, commitTime, hoodieTable,
-            payload.record.getPartitionPath());
+            insertPayload.getPartitionPath());
       }
 
-      if (handle.canWrite(payload.record)) {
+      if (handle.canWrite(((HoodieRecord) payload.getInputPayload()))) {
         // write the payload, if the handle has capacity
-        handle.write(payload.record, payload.insertValue, payload.exception);
+        handle.write(insertPayload,
+            (Optional<IndexedRecord>) payload.getOutputPayload(), payload.exception);
       } else {
         // handle is full.
         statuses.add(handle.close());
         // Need to handle the rejected payload & open new handle
-        handle = new HoodieCreateHandle(hoodieConfig, commitTime, hoodieTable,
-            payload.record.getPartitionPath());
-        handle.write(payload.record, payload.insertValue,
+        handle =
```

Comment:
is this wrapping at 80 or 120? 

`        handle = new HoodieCreateHandle(hoodieConfig, commitTime, hoodieTable, insertPayload.getPartitionPath());`

is only 108 chars.. 

@bvaradar does checkstyle just check for < 120. 

@n3nash if its formatting incorrectly, can you please fix and make a pass on all files again

Reply:
it's formatting incorrectly, I made a pass on all files..